Sources/SymbolKit/SymbolGraph/SymbolGraph+Overloads.swift

```diff
@@ -0,0 +1,126 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+extension SymbolGraph {
+    /// Create "overload group" symbols based on name and kind collisions.
+    ///
+    /// For this method, an "overload" is a symbol whose ``Symbol/pathComponents`` and
+    /// ``Symbol/kind-swift.property`` are the same as another symbol in the same symbol graph.
```

Comment:
this disambiguation is redundant

```suggestion
    /// ``Symbol/kind`` are the same as another symbol in the same symbol graph.
```